CHANGELOG.md

```diff
@@ -1,7 +1,37 @@
 ## Parse Server Changelog
 
 ### master
-[Full Changelog](https://github.com/ParsePlatform/parse-server/compare/2.6.2...master)
+[Full Changelog](https://github.com/ParsePlatform/parse-server/compare/2.6.3...master)
+
+### 2.6.3
+[Full Changelog](https://github.com/ParsePlatform/parse-server/compare/2.6.2...2.6.3)
+
+#### Improvements:
+* Queries on Pointer fields with $in and $nin now supports list of objectId's, thanks to [Florent Vilmart](https://github.com/flovilmart)
+* LiveQueries on $in and $nin for pointer fields work as expected thanks to [Florent Vilmart](https://github.com/flovilmart)
+* Also remove device token when APNS error is BadDeviceToken, thanks to [Mauricio Tollin](https://github.com/)
+* LRU cache is not available on the ParseServer object, thanks to [Tyler Brock](https://github.com/tbrock)
+* Error messages are more expressive, thanks to [Tyler Brock](https://github.com/tbrock)
+* Postgres: Properly handle undefined field values, thanks to [Diamond Lewis](https://github.com/dlewis)
+* Updating with two GeoPoints fails correctly, thanks to [Anthony Mosca](https://github.com/aontas)
+
+#### New Features:
+* Adds ability to set a maxLimit on server configuration for queries, thanks to [Chris Norris](https://github.com/)
+
+#### Bug fixes:
+* Fixes issue affecting reporting _PushStatus with misconfigured serverURL, thanks to [Florent Vilmart](https://github.com/flovilmart)
```

Comment:
hm, is the _ in PushSatus making the rest of it itals??  maybe take out that _

Reply:
yep you're right!

Reply:
according to the interwens, you can escape with \_boo

Reply:
ha \\_boo

Reply:
added backquotes as it looks like code  :)